cmd/ltctl/report.go

```diff
@@ -70,13 +70,21 @@ func RunGenerateReportCmdF(cmd *cobra.Command, args []string) error {
 	}
 	defer f.Close()
 
-	t := terraform.New(nil)
-	output, err := t.Output()
+	promHost, err := cmd.Flags().GetString("prometheus-host")
 	if err != nil {
-		return fmt.Errorf("could not parse output: %w", err)
+		return err
+	}
+
+	if promHost == "" {
+		t := terraform.New(nil)
+		output, err := t.Output()
+		if err != nil {
+			return fmt.Errorf("could not parse output: %w", err)
+		}
+		promHost = output.MetricsServer.Value[0].PublicIP
 	}
 
-	helper, err := prometheus.NewHelper("http://" + output.MetricsServer.Value[0].PublicIP + ":9090")
+	helper, err := prometheus.NewHelper("http://" + promHost + ":9090")
```

Comment:
As a user, I'd probably prefer to be passing the full URL rather the the hostname. Prometheus might be running on `https` or on a different port.